scripts/Muon/GUI/Common/muon_pair.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import (absolute_import, division, print_function)
+
+from Muon.GUI.Common.muon_workspace import MuonWorkspace
+
+
+class MuonPair(object):
+    """Simple struct to store information on a detector group pair.
+
+    The name is set at initialization and after that cannot be changed.
+    The detector list can be modified by passing a list of ints (type checks for this)
+    The number of detetors is stored
+    """
+
+    def __init__(self, pair_name, group1_name="", group2_name="", alpha=1.0):
+        self._pair_name = pair_name
+        self._group1_name = group1_name
```

Comment:
instead of `group1` etc. I think we should signify if it is forwards or backwards e.g. `F_group` or `forward_group`

Reply:
I think `forward_group` because it complies with PEP8 (whereas the capital `F` would possibly give flake8 warnings).

I had felt that using scientific nomenclature inside core code might be confusing to developers who look at the code in the future (your suggestion makes more sense to a scientist who deal with "forward" and "backward"), whereas using an "indexed" notation (i.e. `group_1` and `group_2`) would create less confusion.

I will change to `forward` and `backward`; be aware there may be other places in the core code that also need modifying.